app/values/github_session.rb

```diff
@@ -44,6 +44,19 @@ def get_team_members(team)
     end
   end
 
+  def fetch_teams_for_user(github_login)
```

Comment:
Unused method argument - github_login. If it's necessary, use _ or _github_login as an argument name to indicate that it won't be used. You can also write as fetch_teams_for_user(*) if you want the method to accept any arguments but don't care about them.